host_core/test/host_core/actors_test.exs

```diff
@@ -211,11 +202,11 @@ defmodule HostCore.ActorsTest do
         @httpserver_key,
         @httpserver_contract,
         @httpserver_link,
-        "HandleRequest",
+        "HttpServer.HandleRequest",
```

Comment:
cc @stevelr here, this was required in order to get the actor to handle the request. Is this required, or should the actor also handle `HandleRequest` on it's own?

Previously we'd invoke the actor with `wash ctl call <pubkey> HandleRequest <payload>`, I think the `HttpServer.` prefix makes this a little clunkier, but I want to make sure this is the expected behavior here.